src/flag_gems/ops/mse_loss.py

```diff
@@ -0,0 +1,94 @@
+import logging
+import math
+from enum import Enum
+
+import torch
+import triton
+import triton.language as tl
+
+from ..runtime import torch_device_fn
+from ..utils import libentry, pointwise_dynamic
+from ..utils import triton_lang_extension as tle
+
+
+@libentry()
+@triton.jit
+def kernel_1(inp, target, mid, M, BLOCK_SIZE: tl.constexpr, reduction: tl.constexpr):
+    if tl.constexpr(inp.dtype.element_ty == tl.float16) or tl.constexpr(
+        inp.dtype.element_ty == tl.bfloat16
```

Comment:
dtype of input tensor will be fixed as constant when compiling. it's not necessary to specify it explicitly.